app/src/main/java/org/oppia/app/player/exploration/ExplorationManagerFragment.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.player.exploration
+
+import android.content.Context
+import android.os.Bundle
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/**
+ * FragmentManager of [ExplorationFragment] that observes data provider that retrive default story text size.
+ */
+class ExplorationManagerFragment : InjectableFragment() {
+  @Inject lateinit var explorationManagerFragmentPresenter: ExplorationManagerFragmentPresenter
+
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    fragmentComponent.inject(this)
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    val internalProfileId =
+      arguments!!.getInt(ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, -1)
```

Comment:
Nit: `arguments!!.getInt(ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, /* defaultValue= */ -1)`

Reply:
As this is taking multiple lines, comment should be inline of its value:
```
      arguments!!.getInt(
        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, 
        /* defaultValue= */ -1
      )
```

Reply:
It's already updated in last PR...

Reply:
Actually the code is showing the inline comment in upper line and it should actually be in line of the value:

Before:
```
      arguments!!.getInt(
        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, /* defaultValue= */ 
        -1
      )
```

After:
```
      arguments!!.getInt(
        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, 
        /* defaultValue= */ -1
      )
```

Reply:
Done.